parso/grammar.py

```diff
@@ -1,9 +1,12 @@
 import hashlib
 import os
+from typing import Generic, TypeVar, Union, Dict, Optional, Any
+from pathlib import Path
 
-from parso._compatibility import FileNotFoundError, is_pypy
+from parso._compatibility import is_pypy
 from parso.pgen2 import generate_grammar
-from parso.utils import split_lines, python_bytes_to_unicode, parse_version_string
+from parso.utils import split_lines, python_bytes_to_unicode, \
+    PythonVersionInfo, parse_version_string
```

Comment:
Is it a style preference for you to use backslashes instead of indented groups like
```
from x import (
    y,
    z,
    q
)
```

Reply:
I personally prefer backslashes, but I think it doesn't matter at all, so feel free to use indented groups as well.